web/src/constants/cityNotCooperatingTemplate.ts

```diff
@@ -0,0 +1,2 @@
+export const template =
+  'Sehr geehrte Damen und Herren,\n\nbei der Suche nach [ORT] in der INTEGREAT-App habe ich leider keinen Treffer bekommen. Ich wohne seit  einiger Zeit in Ihrer Kommune und Ich finde es hilfreich für mich und andere, wenn Sie auch bei Integreat mitmachen.\nIn der App finde ich in verschiedenen Sprachen viele wichtige Informationen für meinen Alltag und über die Region. Ich finde es hilfreich für mich und andere, wenn Sie auch bei Integreat mitmachen.\n\nUnter der Homepage: https://integreat-app.de/ erfahren Sie mehr. \n\nHerzliche Grüße'
```

Comment:
is there a reason why we don't put this in translations?

Reply:
yes it should always be in german, as the cities who get these text don't speak arabic or hungarian.